documentation/install.rst

```diff
@@ -121,5 +123,17 @@ To build ``binder`` run
 
 ``make install``
 
-The location of LLVM and CLANG can be set separately with ``-DClang_DIR`` and ``-DLLVM_DIR`` options passed to ``cmake``.
+To perform the build with a specific version of LLVM, the location of LLVM and CLANG directories 
+should be set simultaneously via the location of their cmake configurations, i.e.
+
+``cmake CMakeLists.txt   -DLLVM_DIR=/usr/lib64/llvm8.0/lib/cmake/llvm -DClang_DIR=/usr/lib64/llvm8.0/lib/cmake/clang
+
+Alternatively,the location of the llvm-config script could be set.
+
+``cmake CMakeLists.txt   -DLLVMCONFIG=/usr/lib64/llvm7.0/bin/llvm-config``
+
+By default cmake will build some tests and use python for some simple tests.
+To disable tests use  ``-DBINDER_ENABLE_TEST=OFF``. To suggest python versions to perform tests with use options like
+``-BINDER_TEST_PYTHON_VERSIONS=2,3.5``.  Note that version 0 can be used to check the generated sources agains the reference sources.
+
```

Comment:
Could you please add separate chapter (probably `misc`?) at the end of documentation and outline steps needed to be taken to integrate with GH CI?

Reply:
So far in the updated MR the Ci is split from the support of multiple LLVM versions. So that would make sense to add it in a separate MR.

Reply:
Would make sense to split the merge requests related to Ci as follows:

 - A  MR with CI that checks only if binder compiler. That is the setup from here
 https://github.com/andriish/binderCI. Optionally more OS or compilers can be added.
- A MR that implements a testing suite with ctest.  This is partially implemented in #96 
- A MR that brings the ctest testing to the CI.